test/step-definitions/responses.ts

```diff
@@ -163,4 +163,12 @@ export class responses {
       }
     });
   }
+
+  @then(/the response should contain the Version/)
```

Comment:
I think this should be more generic so we can reuse it

```suggestion
  @then(/the response should contain "([^"]*)"/)
```

Reply:
And what should we put into the brackets when running the test? I think hardcoding a version-number wont work, because we have to adjust the test everytime then, and I'm not sure if I'm a fan putting a regex into the actual feature-file

Reply:
Yep I agree, a regex in the feature file is not ideal